packages/react-components/.storybook/preview.js

```diff
@@ -1,6 +1,7 @@
 import '../src/themes/legacy.scss';
 import '../src/themes/light.scss';
 import '../src/themes/dark.scss';
+import '../src/spacing/spacing.scss';
```

Comment:
Do we need `scss` at all? Using `CSS variables` feels convenient

Reply:
Ok, nevermind it's just an extension of file, it still uses CSS vars

Reply:
👍